programs/cleos/httpc.hpp

```diff
@@ -30,7 +30,7 @@ namespace eosio { namespace client { namespace http {
    parsed_url parse_url( const string& server_url );
 
    fc::variant do_http_call( const connection_param& cp,
-                             const fc::variant& postdata = fc::variant() );
+                             const fc::variant& postdata = fc::variant(), bool verbose = false );
```

Comment:
verbose is too generic, I would use `print_request` here as well.

Reply:
No real reason for that. Just following a common practice - write real output to STDOUT and all diagnostic/debug message to STDERR. So, when you redirect output to file, you still see diagnostic messages in console and your file still contains valid output not mixed with the messages.